### PR TITLE
[FINERACT-1678] Loan accounts stayed locked business event

### PR DESCRIPTION
**Pull request by taskain7**
## Description

We raise a business event at the ond of the Loan COB with the loan IDs that styed locked.

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanAccountStayedLocketBusinessEvent.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.cob.loan;
+
+import java.util.List;
+import org.apache.fineract.infrastructure.event.business.domain.AbstractBusinessEvent;
+
+public class LoanAccountStayedLocketBusinessEvent extends AbstractBusinessEvent<List<Long>> {
```
Comment (adamsaghy) [inline fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanAccountStayedLocketBusinessEvent.java]: typo in the class name: "Locket"

File: fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanCOBManagerConfiguration.java [hunk 2]
```diff
@@ -87,17 +90,28 @@ public Step fetchAndLockStep() {
         return localStepBuilderFactory.get("Fetch and Lock loan accounts - Step").tasklet(fetchAndLockLoanTasklet()).build();
     }
 
+    @Bean
+    public Step stayedLockedStep() {
+        return localStepBuilderFactory.get("Stayed locket loan accounts - Step").tasklet(stayedLockedTasklet()).build();
```
Comment (adamsaghy) [inline fineract-provider/src/main/java/org/apache/fineract/cob/loan/LoanCOBManagerConfiguration.java]: typo in the description: "locket"

**Review by adamsaghy — APPROVED** [review]
LGTM

**Comment by vidakovic** [comment]
not sure what the current rules are with the business event stuff... just putting it out there:
- suffix Event.java => move to package "org.apache.fineract.xxx.event"
- suffix Configuration.java => move to package "org.apache.fineract.xxx.configuration"
- suffix Tasklet.java => move to package "org.apache.fineract.xxx.task"

... maybe for a later refactoring session... will approve.

**Review by vidakovic — APPROVED** [review]
LGTM

**Pull request merged**